systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/ManualRollingUpdateST.java

```diff
@@ -144,6 +145,50 @@ void testManualTriggeringRollingUpdate() {
         assertThat(received, is(sent));
     }
 
+    @Test
+    void testRollingUpdateOnNextReconciliationAfterClusterCAKeyDel() {
```

Comment:
This scenario is a little niche, might it be useful to add a documented comment?

Also might we want to also unit test this scenario?

Reply:
Yeah, I can add it. 

I think the second part is for @scholzj

Reply:
I guess everything can be unit tested ... But real E2E test has probably more value than something full of mocking etc. which is how the unit test would end. So I think we should definitely keep this ST and if someone writes UT as well, it would be just a bonus.

Reply:
I added the scenario as a comment, thanks :)